src/main/java/org/spongepowered/common/mixin/core/entity/item/EntityItemMixin.java

```diff
@@ -192,6 +192,21 @@
             ci.cancel();
         }
     }
+    
+    @Inject(
+        method = "onUpdate", 
+        at = @At (
+            value = "INVOKE", 
+            ordinal = 1, 
+            target = "Lnet/minecraft/entity/item/EntityItem;setDead()V")
+    )
+    private void spongeImpl$fireExpireEntityEventTargetItem(final CallbackInfo ci) {
```

Comment:
Not to be too nit picky:
`impl$fireExpireEntityEvent` is what the method name should be. Prefix is `impl$` not `spongeImpl$`